### PR TITLE
Added Tests for node js samples

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
Added Tests for node js samples

### CODE REVIEW

File: node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const process = require('process');
+const webdriver = require('selenium-webdriver');
+const {until} = require('selenium-webdriver');
+
+process.env['Path'] = process.env['Path'] + ';' + process.env['ChromeWebDriver'];
```
Comment (acesiddhu) [inline node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js]: how are we getting ChromeDriver to that folder?

Comment (ShreyasRmsft) [reply node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js]: it is already there on the hosted machines

File: node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		driver.quit()
+			.then(() => done())
+			.catch(() => {
+				done();
+			});
+    });
+
+    it('Assert page title', (done) => {
+		driver.get(process.env['webAppUrl']).then(() => {
+			return driver.wait(until.titleIs('Microsoft Azure App Service - Welcome'), 2000);
```
Comment (ShreyasRmsft) [inline node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js]: check if this needs to be changed

**Comment by ShreyasRmsft** [comment]
changes pending for nodejs with container. do not merge yet

File: node/sail.js/containerWithTests/Application/Application/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM node:6.9.3
+LABEL maintainer="Azure App Service Container Images <appsvc-images@microsoft.com>"
+
+# Create app directory
+WORKDIR /app
+
+# Install app dependencies
+COPY package.json .
```
Comment (bishal-pdMSFT) [inline node/sail.js/containerWithTests/Application/Application/Dockerfile]: remove npm install

**Pull request merged**